src/ISA/ISA/ArcTypes/ArcTable.fs

```diff
@@ -712,7 +713,19 @@ type ArcTable(name: string, headers: ResizeArray<CompositeHeader>, values: Syste
 
     // it's good practice to ensure that this behaves using the same fields as Equals does:
     override this.GetHashCode() = 
-        let name = this.Name.GetHashCode()
-        let headers = this.Headers |> Seq.fold (fun state ele -> state + ele.GetHashCode()) 0
-        let bodyCells = this.Values |> Seq.sortBy (fun x -> x.Key) |> Seq.fold (fun state ele -> state + ele.GetHashCode()) 0
-        name + headers + bodyCells
\ No newline at end of file
+        //let v1,v2 = 
+        let v =
```

Comment:
hashcode does ignore order?
If so, does equality behave the same way?

Reply:
i have a very nice test for table equality, but unsure how do good tests for inequality on assay,study,investigation. Have a look